connector/profilingmetricsconnector/frame.go

```diff
@@ -115,36 +115,60 @@ var (
 	// TODO: CNIs like Cilium may introduce complications and the following list
 	// may need to be updated.
 	classes = []class{
-		{name: "network/tcp/read",
-			rx: regexp.MustCompile(`^(?:tcp_recvmsg|tcp_read|tcp_rcv|tcp_v4_rcv|tcp_v6_rcv)`)},
-		{name: "network/tcp/write",
-			rx: regexp.MustCompile(`^(?:tcp_sendmsg|tcp_transmit|tcp_write|tcp_push|tcp_sendpage)`)},
-		{name: "network/udp/read",
-			rx: regexp.MustCompile(`^(?:udp_recvmsg|udp_read|udp_rcv|udp_v4_rcv|udp_v6_rcv)`)},
-		{name: "network/udp/write",
-			rx: regexp.MustCompile(`^(?:udp_sendmsg|udp_write|udp_push|udp_sendpage)`)},
-		{name: "ipc/read",
+		{
```

Comment:
Maybe not for this PR, but can we add a `Makefile` target to perform this formatting? Running `gofmt` doesn't match this formatting and we need something that's reproducible and consistent across various editors. It will also speed up reviews (less lines of modified code to be checked).